REPOSITORIES.md

```diff
@@ -0,0 +1,26 @@
+# KubeStellar Repository Ecosystem
+
+KubeStellar is composed of multiple repositories that together enable robust multi-cluster configuration management. Below is a summary of each codebase and its primary purpose.
+
+## Core Platform
+- **kubestellar/kubestellar** — The heart of KubeStellar: a Go-based controller and API server that reconciles multi-cluster resource bindings across any number of Kubernetes clusters.
+- **kubestellar/kubeflex** — A pluggable control-plane implementation, written in Go, that lets you run Kubernetes APIs on demand for any cluster, edge node, or environment.
+
+## Transport & Integration
+- **kubestellar/ocm-status-addon** — A Go add-on that integrates with the Open Cluster Management (OCM) transport layer to surface multi-cluster status in OCM dashboards.
```

Comment:
I would put this in the same category as kubestellar/kubestellar, because part of the OCM-based transport is in kubestellar/kubestellar.